### ISSUE TITLE
Update Tabs Component to  match core

### ISSUE THREAD

**Issue by tlabaj** [patternfly/patternfly-react#1268]
Update the Tab Component to match core once issue [#1328](https://github.com/patternfly/patternfly-next/issues/1328) has been resolved.

> when a set of tabs is totally unique on the page they should be within a <nav> instead so that screen reader users can find them by browsing for navigational landmarks. The HTML would then be:

**Comment by rachael-phillips** [comment]
HI @rebeccaalpert would this be something you could incorporate on what you're working on right now? Let me know what you think about this!

**Comment by rebeccaalpert** [comment]
Yeah, that should be doable. Is it still needed, though? It looks like https://github.com/patternfly/patternfly-next/issues/1328 was closed rather than merged.
